src/signals-win.c

```diff
@@ -278,59 +298,88 @@ LONG WINAPI jl_exception_handler(struct _EXCEPTION_POINTERS *ExceptionInfo)
             break;
         }
     }
+    ios_t full_error, summary;
+    ios_mem(&full_error, 1024);
```

Comment:
calling malloc unconditionally here is a bit wasteful, since we will call it on demand when actually needed
```suggestion
    ios_mem(&full_error, 0);
```